README.md

```diff
@@ -2,6 +2,46 @@
 
 Markcop is our friendly Markdown enforcer.
 
+## Contributing
+
+This is a guide to creating your own check.
+
+1. Add a function that takes the file name as the its first argument, and the
+   file contents as the second
+
+   ```bash
+   function my_check {
+     file_name="$1"
+     my_check_errors_in_file=false
+     for line in $(echo "$2"); do
+       my_check_errors_in_file=true
+       # Failed checks should print red 'x's
+       printf "${RED}x${NC}"
+       errors="${errors}\n${line} fails my_check!"
+     done
+
+     if [ $my_check_errors_in_file = true ]; then
+       return 1
+     fi
+
+     # Successful checks should print green dots
+     printf "${GREEN}.${NC}"
+     return 0
+   }
+   ```
+
+2. Add the function name to the `CHECKS` array
+
+   ```
+   CHECKS=(my_check
+           trailing_whitespace
+           malformed_header
+           header_missing_newline
+           long_line
+           missing_link
+           eof_newline)
+   ```
+
 ## Running It
 
 Copy and paste the following into your terminal (must have Docker installed):
```

Comment:
We're using circleCI for deployments to docker. After a check is added to the master branch of hackedu/markcop, it will automatically be deployed to any of our other repos.